utils/hooks/useStats.ts

```diff
@@ -0,0 +1,28 @@
+import useSWR from "swr";
+import { fetcher } from "../fetcher";
+
+export type UseStatsResponse = {
+    totalRegistrants: number;
+    totalPrograms: number;
+    totalClasses: number;
+    totalTeachers: number;
+    isLoading: boolean;
+    error: any;
+    mutate: (data?: any, shouldRevalidate?: boolean) => Promise<any>;
+};
+/**
+ * Use hook for getting stats for admin dashboard
+ * @returns UseStatsResponse
+ */
+export default function useStats(): UseStatsResponse {
+    const { data, error, mutate } = useSWR("/api/admin/stats", fetcher);
+    return {
+        totalRegistrants: data && data.data ? data.data.totalRegistrants : null,
```

Comment:
Thoughts - no action required

I think you can just do `data?.data?.totalRegistrants` in ts instead of `data && data.data ? data.data.totalRegistrants : null`

Additionally, you could map the data to an array and then use the `...mappedData` notation before isLoading.

Reply:
yes facts, that's a really good point. I'll make that change!